Makefile

```diff
@@ -321,20 +321,17 @@ dev-db-openshift: prebuild-check deps generate $(FRESH_BIN)
 
 .PHONY: dev-openshift
 dev-openshift: prebuild-check deps generate build bin/docker/fabric8-auth-linux
-	minishift start --cpus 4
+	minishift start --cpus 3
 	./minishift/check_hosts.sh
-	-eval `minishift oc-env` &&  oc login -u developer -p developer && oc new-project auth-openshift
-	AUTH_DEVELOPER_MODE_ENABLED=true \
-	kedge apply -f minishift/kedge/db-auth.yml
-	sleep 5s
-	-eval `minishift docker-env` && docker login -u developer -p $$(oc whoami -t) $$(minishift openshift registry) && docker build -t fabric8/fabric8-auth:dev bin/docker
```

Comment:
we don't need to login to internal docker registry so removed it

Reply:
How can we build an image from compiled local source in the internal docker registry if we don't connect to it?

Reply:
so when you do `eval ` on minishift docker-env you already have access to the docker environment of the minishift machine, so when you do `docker build ` it is happening inside the VM so the image is already available to the openshift cluster inside minishift so no need to login to the registry.

But if you want to use the registry just logging in is not enough because you will also have to push the image to the registry and then use it.

Reply:
Thanks for the clarification, I thought the docker login was necessary to be able to build the new docker image inside minishift's docker registry, but if setting the env variables is sufficient then that is fine.